sdk/src/test/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessorTest.java

```diff
@@ -329,14 +126,14 @@ public void exportDifferentSampledSpans() {
   @Test
   public void exportMoreSpansThanTheBufferSize() {
     WaitingSpanExporter waitingSpanExporter = new WaitingSpanExporter(6);
-    BatchSpansProcessor.Config config =
-        BatchSpansProcessor.Config.newBuilder()
+    BatchSpansProcessor bsp =
```

Comment:
would prefer not abbreviating like this. We should prefer spelling out names to make reading the call sites less confusing.

Reply:
To rephrase what @jkwatson said: we are Java not Go :)))

Reply:
Addressed in 0af5490